Phase 3

electricg opened the pull request:
- Adjusted schedule time
- Added speaker photo urls
- Added description to breaks
- Small tweaks

remy commented on src/components/sessions/data.js:
```diff
@@ -56,8 +58,8 @@ module.exports = [
-    duration: 30,
-    title: 'Cake break (30 mins)',
+    duration: 25,
+    title: 'Cake break (25 mins)',
```
Can't tell from the diff, but is this the afternoon break? If so, let's change `cake` to `ice cream`.

electricg replied:
Yes this is the afternoon break. I'll change it to "ice cream" then 👍

electricg replied:
Fixed

electricg commented:
- Added the list of files for service workers (but I did **not** activate it, to activate it, you need to change the relative flag in the config file to true)
- Changed the implementation of the quote icon, otherwise it was not working with service workers on offline mode
